charts/smokey/values.yaml

```
@@ -5,6 +5,13 @@ publishingDomainSuffix: test.publishing.service.gov.uk
 assetsDomain: assets.test.publishing.service.gov.uk
 
 cronSchedule: "*/10 * * * *"
+smokeyCronTasks:
```

The `smokey` prefix is redundant. (Also "jobs" might be more intuitive, since each represents a cronjob?)
```suggestion
cronJobs:
```